pages/patient-info.tsx

```diff
@@ -68,6 +71,7 @@ export default function PatientInfoPage() {
           <BaseInput
             label="Phone number"
             type="number"
+            defaultValue={userData?.info?.phone}
```

Comment:
this is definitely not correct. We're using react hook forms which means you should be using the initial values of the library instead of defining a defaultValue here. DefaultValue is a property that normally is used only for uncontrolled components and react hook forms is always gonna give a value to these BaseInputs, making them controlled